.github/workflows/cli-integration-test.yaml

```diff
@@ -0,0 +1,35 @@
+name: Run CLI integration tests
+
+on: pull_request
+
+jobs:
+  integration-test:
+    runs-on: ubuntu-latest
+    timeout-minutes: 20
+
+    steps:
+    - name: Checkout repository
+      uses: actions/checkout@v2
+
+    - name: Set up Docker
+      uses: docker/setup-buildx-action@v1
+
+    - name: Install Kind
+      run: |
+        curl -Lo ./kind https://kind.sigs.k8s.io/dl/v0.17.0/kind-linux-amd64
+        chmod +x ./kind
+        sudo mv ./kind /usr/local/bin/kind
+
+    - name: Create Kind cluster
+      run: kind create cluster --name integration-tests --wait 2m
+
+    # TODO matrix for testing against different Syntho stack versions TBD later on
+    - name: Run integration tests
+      working-directory: ./cli/tests/integration/
+      env:
+        CLUSTER_NAME: integration-tests
+        VERSION: 1.9.0
```

Comment:
I'll pin this to the latest version. This way of setting the version should be improved upon, but let's start with this for now.